tinygrad/runtime/ops_metal.py

```diff
@@ -41,22 +41,21 @@ def unwrap(x):
   return ret
 
 @diskcache
-def compile_metal(prg, use_xcode=bool(getenv("METAL_XCODE"))) -> bytes:
+def compile_metal(prg, use_xcode=bool(getenv("METAL_XCODE"))):
```

Comment:
Hmm, my issue with this is that all the compile functions have had a very clear function signature, `(str) -> bytes`, this breaks that.

Reply:
is there a specialization equivalent here?